openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-settings.js

```diff
@@ -357,6 +357,11 @@ var VideoSettings = (function() {
 	}
 	function _getDevConstraints(callback) {
 		const devCnts = {audio: false, video: false};
+		if (navigator.mediaDevices === undefined) {
+			OmUtil.error('Brower multimedia devices are blocked by security settings. Try https.');
```

Comment:
There is a typo in this message `/Brower/Browser/`

Do we need this one localized? i.e. shall this string be moved to language file and later displayed in user's locale?

Reply:
I fixed the typo, thanks.

How would you localise it?
I can see many OmUtil.error(error); none of them seem localised.
Is there an example somewhere ?

Reply:
Just found
https://github.com/apache/openmeetings/blob/master/openmeetings-web/src/main/java/org/apache/openmeetings/web/room/VideoSettings.html#L106
This is already localized
And most probably displayed :)))

> I can see many OmUtil.error(error); none of them seem localised.
Lazy me :((

Reply:
Localization is here
https://github.com/apache/openmeetings/blob/master/openmeetings-web/src/main/java/org/apache/openmeetings/web/app/Application.properties.xml#L1002
(and in other `Application.properties_*.xml` files)

Reply:
I just added my comment to the localised message.